--- a/codecov.io.sh	2017-12-01 13:04:05.075361774 +0100
+++ b/codecov.io.sh	2017-12-01 13:04:14.835362239 +0100
@@ -1116,6 +1116,8 @@
                     -not -name '*.db' \
                     -not -name '*.md' \
                     -not -name '*.cpp' \
+                    -not -name '*.x' \
+                    -not -name '*.f' \
                     -not -name '*.gradle' \
                     -not -name '*.tar.tz' \
                     -not -name '*.scss' \
@@ -1349,7 +1351,7 @@
       || echo ''
   fi
 
-  if echo "$network" | grep -m1 '\(.cpp\|.h\|.cxx\|.c\|.hpp\|.m\)$' 1>/dev/null;
+  if echo "$network" | grep -m1 '\(.cpp\|.h\|.cxx\|.c\|.x\|.f\|.hpp\|.m\)$' 1>/dev/null;
   then
     # skip brackets
     find "$git_root" -type f \
@@ -1360,6 +1362,8 @@
            -or -name '*.cxx' \
            -or -name '*.m' \
            -or -name '*.c' \
+           -or -name '*.x' \
+           -or -name '*.f' \
            -or -name '*.hpp' \
          \) -exec \
       grep -nIHE \
@@ -1380,6 +1384,8 @@
            -or -name '*.cxx' \
            -or -name '*.m' \
            -or -name '*.c' \
+           -or -name '*.x' \
+           -or -name '*.f' \
            -or -name '*.hpp' \
          \) -exec \
       grep -nIH '// LCOV_EXCL' \
